test/runtests.jl

```diff
@@ -0,0 +1,20 @@
+using WinRPM
+using Test
+
+@testset "WinRPM.jl" begin
+    @testset "WinRPM.update" begin
+        for (ignorecache, allow_remote) in [(false, true), (true, true)]
+            WinRPM.update(ignorecache, allow_remote)
+        end
+    end
+
+    if Sys.iswindows()
```

Comment:
This globally installs bzip2 for everyone running the test suite?

Shouldn't this then remove it afterwards.

Also if the user already has this package installed we should be careful not to modify it or remove it?

Reply:
Maybe we should only run this test on CI.

Reply:
I think it'll be too complicated to figure out how to "cleanly" do this in a user's local computer.

Reply:
Yes, I think we should only run the tests on CI.

Reply:
Is it possible to configure that?

Reply:
Yeah, give me one second.

Reply:
Done.